map/src/menu/favorite/FavoritesMenu.jsx

```diff
@@ -20,7 +21,11 @@ export default function FavoritesMenu() {
     const groupItems = useMemo(() => {
         const items = [];
         const groups =
-            sortGroups?.length > 0 ? sortGroups : ctx.favorites?.groups?.length > 0 ? ctx.favorites?.groups : null;
+            sortGroups?.length > 0
+                ? sortGroups
+                : ctx.favorites?.groups?.length > 0
+                ? byTime(ctx.favorites?.groups, true, true)
+                : null;
```

Comment:
Almost unreadable code `... ? : ? : ...`

Please avoid using nested ternaries or try experimental prettier feature for better formatting:

https://prettier.io/blog/2023/11/13/curious-ternaries.html

https://www.sonarsource.com/blog/stop-nesting-ternaries-javascript/